src/components/zine-card.tsx

```diff
@@ -0,0 +1,54 @@
+import Image from "next/image";
+import Link from "next/link";
+import { getThumbnailUrl } from "@/utils/assets";
+
+type ZineCardProps = {
+  zine: Zine;
+};
+
+const ZineCard: React.FC<ZineCardProps> = ({ zine }) => {
+  return (
+    <div
+      key={zine.uuid}
```

Comment:
You don't need to pass the key here. It's already applied when you add a key in the parent file.

```suggestion
    <div
```

Reply:
great catch!